test/test_config/missing_machine_netrc

```diff
@@ -4,4 +4,4 @@ machine https://randommachine/
 
 machine https://anotherrandommachine
   login someadmin
-  password conjurapikey
+  password Apikey
```

Comment:
why?

Reply:
it was chanhged by mistake with ctrl shift r . Good Catch. I take it back